controls/templatedbutton/visual-structure.md

```diff
@@ -8,27 +8,22 @@ slug: templatedbutton-visual-structure
 
 # .NET MAUI TemplatedButton Visual Structure
 
-The visual structure of the .NET MAUI TemplatedButton represents the anatomy of the UI component. Being familiar with the visual elements of the Calendar allows you to quickly find the information required to configure them.
+The visual structure of the .NET MAUI TemplatedButton represents the anatomy of the UI component. Being familiar with the visual elements of the TemplatedButton allows you to quickly find the information required to configure them.
 
 The following image shows the anatomy of the TemplatedButton.
 
-![.NET MAUI TemplatedButton Visual Structure](images/templatedbutton-visual-structure.png "Visual elements of TemplatedButton control")
+![.NET MAUI TemplatedButton Visual Structure](images/templatedbutton-visual-structure.png "Visual elements of The TemplatedButton control")
 
 ## Displayed Elements
 
-## Additional Resources
+* **Content**&mdash;The content of the TemplatedButton.
```

Comment:
Check out our very own design system and how it discusses the anatomy of a button: https://www.telerik.com/design-system/docs/components/button/ 

Consider talking about "container" and "content". (You can do this in a future version of the doc set though and merge as is.)